### PR TITLE
Point to API call the fetch extension settings ...

### PR DESCRIPTION
**Pull request by kaystrobach**

### CODE REVIEW

**Comment by kaystrobach** [comment]
wow that was fast 😄

**Comment by marble** [comment]
Yes, I was just working on this kind of stuff :-)) Something to backport?

**Comment by kaystrobach** [comment]
i think this worked since 6.2, but I'm not totally sure

**Comment by marble** [comment]
Applied to 'latest' and '7-6'

**Comment by kaystrobach** [comment]
i'll check with the api docs

**Comment by kaystrobach** [comment]
it's in the docs for 6.2 as well:

* http://api.typo3.org/typo3cms/62/html/class_t_y_p_o3_1_1_c_m_s_1_1_extensionmanager_1_1_utility_1_1_configuration_utility.html#ab4a31e030e2c351f1e0148451f11840c

**Comment by marble** [comment]
Fine, thanks!

File: Documentation/ExtensionArchitecture/ConfigurationOptions/Index.rst [hunk 1]
```diff
@@ -48,6 +48,14 @@ you could simply use::
    $extensionConfiguration = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['your_extension_key']);
    $temporaryDirectory = $extensionConfiguration['temporaryDirectory'];
 
+Or even better use the API to get the information merged with the defautl settings,
```
Comment (romaincanon) [inline Documentation/ExtensionArchitecture/ConfigurationOptions/Index.rst]: defau**tl** → defau**lt** @kaystrobach :)

Comment (marble) [inline Documentation/ExtensionArchitecture/ConfigurationOptions/Index.rst]: Is already fixed in the final merge.

Comment (romaincanon) [inline Documentation/ExtensionArchitecture/ConfigurationOptions/Index.rst]: Oops, my bad!

Comment (marble) [inline Documentation/ExtensionArchitecture/ConfigurationOptions/Index.rst]: No, your good, great you spotted it!

**Pull request merged**